code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -732,30 +760,135 @@
 	icon_state = "medical"
 	access = list(ACCESS_MEDICAL, ACCESS_MORGUE, ACCESS_SURGERY, ACCESS_CHEMISTRY, ACCESS_VIROLOGY, ACCESS_GENETICS, ACCESS_MINERAL_STOREROOM)
 
+/obj/item/card/id/coroner
+	name = "Coroner ID"
+	registered_name = "Coroner"
+	icon_state = "coroner"
+	access = list(ACCESS_MEDICAL, ACCESS_MORGUE, ACCESS_SURGERY, ACCESS_CHEMISTRY, ACCESS_VIROLOGY, ACCESS_GENETICS, ACCESS_MINERAL_STOREROOM)
```

Comment:
Coroner has ORM access? This should probably be removed

Reply:
these are copy pastes of the default access the job gets

uhhhHHH make another PR to remove the access? I am not changing it in this PR, very unrelated